examples/oft-hyperliquid/layerzero.config.ts

```diff
@@ -1,44 +1,53 @@
 import { EndpointId } from '@layerzerolabs/lz-definitions'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities'
+import { TwoWayConfig, generateConnectionsConfig } from '@layerzerolabs/metadata-tools'
+import { OAppEnforcedOption } from '@layerzerolabs/toolbox-hardhat'
 
-import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+import type { OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
 
-const bscTestnetContract: OmniPointHardhat = {
-    eid: EndpointId.ARBSEP_V2_TESTNET,
-    contractName: 'MyOFT',
+const hyperevmContract: OmniPointHardhat = {
+    eid: EndpointId.HYPERLIQUID_V2_TESTNET,
+    contractName: 'MyOFT', // Use 'MyOFT' for both testnet and production
 }
 
-const hyperliquidTestnetContract: OmniPointHardhat = {
-    eid: EndpointId.HYPERLIQUID_V2_TESTNET,
-    contractName: 'MyOFT',
+const arbitrumContract: OmniPointHardhat = {
+    eid: EndpointId.ARBSEP_V2_TESTNET,
+    contractName: 'MyOFT', // Use 'MyOFT' for both testnet and production
 }
 
-const config: OAppOmniGraphHardhat = {
-    contracts: [
-        {
-            contract: bscTestnetContract,
-            config: {
-                owner: '',
-                delegate: '',
-            },
-        },
-        {
-            contract: hyperliquidTestnetContract,
-            config: {
-                delegate: '',
-                owner: '',
-            },
-        },
-    ],
-    connections: [
-        {
-            from: bscTestnetContract,
-            to: hyperliquidTestnetContract,
-        },
-        {
-            from: hyperliquidTestnetContract,
-            to: bscTestnetContract,
-        },
+// To connect all the above chains to each other, we need the following pathways:
+// Base <-> Arbitrum
+
+// For this example's simplicity, we will use the same enforced options values for sending to all chains
+// For production, you should ensure `gas` is set to the correct value through profiling the gas usage of calling OFT._lzReceive(...) on the destination chain
+// To learn more, read https://docs.layerzero.network/v2/concepts/applications/oapp-standard#execution-options-and-enforced-settings
+const EVM_ENFORCED_OPTIONS: OAppEnforcedOption[] = [
+    {
+        msgType: 1,
+        optionType: ExecutorOptionType.LZ_RECEIVE,
+        gas: 80000,
+        value: 0,
+    },
+]
+
+// With the config generator, pathways declared are automatically bidirectional
+// i.e. if you declare A,B there's no need to declare B,A
+const pathways: TwoWayConfig[] = [
+    [
+        hyperevmContract, // Chain A contract
+        arbitrumContract, // Chain B contract
+        [['LayerZero Labs'], []], // [ requiredDVN[], [ optionalDVN[], threshold ] ]
+        [1, 15], // [A to B confirmations, B to A confirmations]
```

Comment:
Arbitrum default confirmations are 20.

```suggestion
        [1, 20], // [A to B confirmations, B to A confirmations]
```

Reply:
updated